pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -583,7 +617,23 @@ func adsPushAll() {
 				}
 			}
 		}
-		//}(client)
+	}
+
+	if s != nil {
+		time.AfterFunc(10*time.Second, func() {
```

Comment:
can we have constant for this?

Reply:
Yes, of course. Hopefully for short time - the proper behavior is to wait until all pushes finish, but I don't want to mess with channels.

Reply:
No I meant can you use a constant declaration with some meaningful name so that others understand what this timer is for?